packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -461,11 +462,37 @@ export default function useAutocomplete(props) {
     },
   );
 
+  const previousProps = usePreviousProps({
+    filteredOptions,
+    value,
+    multiple,
```

Comment:
I don't think we need to store this `multiple` prop to access its previous state.

Reply:
yes you are right , mutiple is not required. removed multiple from usePreviousProps hook